features/toolbar/components/ToolbarContainer.native.js

```diff
@@ -22,25 +23,40 @@ export class ToolbarContainer extends Component {
     render() {
         var underlayColor = ColorPalette.buttonUnderlay;
         var micButtonStyle;
+        var micIconStyle;
         var micButtonIcon;
         if (this.props.audioMuted) {
             micButtonStyle = [
                 styles.toolbarButton,
                 { backgroundColor: underlayColor }
             ];
-            micButtonIcon = 'microphone-slash';
+            micIconStyle = [styles.icon, { color: 'white' }];
+            micButtonIcon = 'mic-disabled';
         }
         else {
             micButtonStyle = styles.toolbarButton;
+            micIconStyle = styles.icon;
             micButtonIcon = 'microphone';
         }
 
+        // The following property is responsible to hide/show the toolbar
+        // view by moving it out of site of the screen boundaries.
+        // An attempt to use the opacity property has been made in order
+        // to eventually implement a fadeIn/fadeOut animation, however a known
+        // react native problem has been doscovered, which allows the view to
+        // still capture touch events even if hidden. Alternatives will be
+        // investigated.
+        var bottomValue = styles.toolbarContainer.bottom;
+        if (!this.props.isVisible)
+            bottomValue = -Dimensions.get('window').height;
```

Comment:
The 3 lines above + the comment associated with them are what I've been asking @legastero & @tsareg to consider as a duplication. I'm basing my observation of a duplication on the following:

- The two pieces of source code represent 2 implementations of 1 'rule'.

- The 2 implementations are obviously unclear enough to warrant comments that describe them. (It's of course OK for an implementation to be difficult to read through & to need a comment that facilitates understanding. But having such a situation in 2 places is worse than having it in 1 place.)

- The 2 implementations are unfinished & have future work planned (as stated by the comments).

- The comment on the filmStrip copy of this code is obviously at least some part of a copy+paste of this instance because it's comment talks about toolbar instead of filmStrip.

Anyway, I'll accept the source code as it is due to the urgency of the merge.